lockbox-ios/Presenter/AccountSettingPresenter.swift

```diff
@@ -5,10 +5,12 @@
 import Foundation
 import RxCocoa
 import RxSwift
+import UIKit
 
 protocol AccountSettingViewProtocol: class {
     func bind(avatarImage: Driver<Data>)
     func bind(displayName: Driver<String>)
+    func displayAlert()
```

Comment:
You may want to look at the `UIViewController+` class -- there is already an implementation of an option sheet that just needs to be extended a little to handle the `Destructive`-style button.  That should also help you get unblocked on binding the Observer to the button action.